packages/dev/docs/src/PropTable.js

```diff
@@ -76,7 +76,12 @@ const GROUPS = {
 };
 
 export function PropTable({component, links, style}) {
-  let [ungrouped, groups] = groupProps(component.props.properties);
+  let ungrouped, groups;
+  if (!component.props) {
```

Comment:
I'm not sure what about the typescript update broke this, but some times we now get 'parameters' instead of just 'props'